wremnants/muon_selections.py

```diff
@@ -67,7 +67,7 @@ def select_z_candidate(df, ptLow, ptHigh):
     return df
 
 def apply_triggermatching_muon(df, dataset, muon_eta, muon_phi):
-    if dataset.group in ["Top", "Diboson"]:
+    if dataset.group in ["Top", "Diboson", "QCD"]:
```

Comment:
Probably a good idea to introduce a variable for this in common (background_mc or something)

Reply:
done btw